app/controllers/platform/api/v1/users_controller.rb

```diff
@@ -7,8 +7,8 @@ class Platform::Api::V1::UsersController < PlatformController
 
   def create
     @resource = (User.find_by(email: user_params[:email]) || User.new(user_params))
+    @resource.skip_confirmation!
     @resource.save!
-    @resource.confirm
```

Comment:
Is it possible that some users of the platform api are relying on this email being sent for whatever reason?
If so maybe this would need a deprecation warning first or a configuration option or something?

Reply:
@jordan-brough I think the earlier behaviour was unintentional. 
Let's go ahead with the merge. We will mention the change in the release notes.